src/actions/adaptive-container.ts

```diff
@@ -28,6 +30,44 @@ export class AdaptiveActionContainer<T extends Action = Action, V extends IActio
       });
   }
 
+  private showFirstN(visibleItemsCount: number) {
+    let leftItemsToShow = visibleItemsCount;
+    const invisibleItems: Action[] = [];
+    this.actions.filter(action => action.visible).forEach((item) => {
+      if (item === this.dotsItem) {
+        return;
+      }
+      if (leftItemsToShow <= 0) {

```

Comment:
Elements that index greater then `visibleItemsCount` carry to `invisibleItems` and its `mode` set to `"popup"`. But there is no actions with elements that index less then `visibleItemsCount`. Is it OK? Anyway method name is unobvious